xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -256,6 +277,19 @@ def _skip_target(*, target, deps, deps_attrs, transitive_infos):
         ),
     )
 
+def _create_test_envs_depset(*, automatic_target_info, ctx, id, target):
+    test_env = getattr(ctx.rule.attr, automatic_target_info.env, {})
+    raw_run_env = target[RunEnvironmentInfo].environment if RunEnvironmentInfo in target else {}
```

Comment:
We need to update `README.md` to say 5.3 is the minimum Bazel version, unless we somehow detect the presence of a global symbol.

@keith @thii do either of you know how we could detect if the `RunEnvironmentInfo` provider is available? Does `hasattr(native, "RunEnvironmentInfo")` work?